python/requirements.txt

```diff
@@ -1,6 +1,6 @@
 async-timeout==4.0.2;python_version<"3.11"
 maturin==0.13.0
-protobuf==3.20.*
+protobuf==4.25.*
```

Comment:
Do we have to `==`? If a user already has 3, are we certain it's a problem? I understood that one user had an issue with conflicts because of it. I'm afraid that if we are strict for no reason, we force users while it's not really needed.

Reply:
Yes, `protoc` version should match runtime lib version. There is no implication for a user.